dali/pipeline/executor/executor_test.cc

```diff
@@ -593,36 +584,12 @@ TYPED_TEST(ExecutorSyncTest, TestPrefetchedExecution) {
   exe->RunMixed();
   exe->RunGPU();
 
-  // Verify that both sets of results are correct
   DeviceWorkspace ws;
   exe->Outputs(&ws);
-  ASSERT_EQ(ws.NumOutput(), 1);
-  ASSERT_EQ(ws.NumInput(), 0);
-  ASSERT_TRUE(ws.OutputIsType<GPUBackend>(0));
-  TensorList<GPUBackend> &res1 = ws.Output<GPUBackend>(0);
-  for (int i = 0; i < batch_size; ++i) {
-    this->VerifyDecode(
-        res1.template tensor<uint8>(i),
-        res1.tensor_shape(i)[0],
-        res1.tensor_shape(i)[1], i);
-  }
+  test::CheckResults(ws, batch_size, 0, tl);
 
   exe->Outputs(&ws);
-  ASSERT_EQ(ws.NumOutput(), 1);
-  ASSERT_EQ(ws.NumInput(), 0);
-  ASSERT_TRUE(ws.OutputIsType<GPUBackend>(0));
-
-  auto status_2 = barrier_future_2.wait_for(std::chrono::seconds(5));
-  ASSERT_EQ(status_2, std::future_status::ready);
-  ASSERT_EQ(cb_counter, 2);
```

Comment:
Should we really get rid of those cb_counter checks? (I don't think it sees much use, but it is still some kind of feature).